docs/installation/localvir.md

```diff
@@ -46,12 +46,18 @@ $ workon badgeyay               # To activate it again
 cp .env.example .env
 ```
 
-* To run the project on a local machine.
+* To run the project on a local machine (default mode).
 
 ```sh
 (badgeyay) $ python app/main.py
 ```
 
+* To run the project on a local machine (debug mode).
+
+```sh
+(badgeyay) $ python app/main.py --dev
```

Comment:
@raju249 What I we should use `python app/main.py` and if they want to start server in debug mode than can add `--dev` in documentation. Your views please

Reply:
Yes you are right. That's why I have added both modes in docs